### PR TITLE
Moderate blood loss now does tapering damage.

### LINKED ISSUE #14538: Critical blood loss knocks you out forever
I'm fine with temporarily falling unsconsious due to blood loss. 
Being in a perma-stun, however, is bad gameplay.

### PR DESCRIPTION
**Pull request by Buggy123**
Fixes #14538 without making everyone heal oxyloss. Although I have no fucking clue why IF( oxyloss < 50) was triggering with more than 50 oxyloss. MSO said he was working on a better solution so this may be temporary.

Edit: MSO came up with it and Paprika helped, credit where credit is due.

### CODE REVIEW

File: code/modules/mob/living/carbon/human/blood.dm [hunk 1]
```diff
@@ -83,15 +83,12 @@ var/const/BLOOD_VOLUME_SURVIVE = 122
...
-					oxyloss += 3
+				oxyloss += max(0, (20-oxyloss)*0.2) // multiply don't divide, if you want a good ride
 			if(BLOOD_VOLUME_BAD to BLOOD_VOLUME_OKAY)
 				if(!pale)
 					pale = 1
 					update_body()
-				if(oxyloss < 50)
-					oxyloss += 10
-				oxyloss += 1
+				oxyloss += max(1, (50-oxyloss)*0.2) //multiply don't divide, if you want to avoid a tide(of LAG)
```
Comment (tkdrg) [inline code/modules/mob/living/carbon/human/blood.dm]: These comments are pretty unneeded.
Division isn't even that slow anyway.

**Comment by optimumtact** [comment]
:+1: assuming the comments are removed

**Comment by Buggy123** [comment]
>implying there aren't a ton of silly comments in the code.

**Comment by tkdrg** [comment]
Do you know how to squash?

**Comment by Buggy123** [comment]
Uh... lemme look it up and boot up my ghetto-puter.

**Comment by Buggy123** [comment]
Wew

**Comment by tkdrg** [comment]
Nice job. You could have amended the commit message to be something a bit more descriptive but this is acceptable.

**Comment by Buggy123** [comment]
Weird, shouldn't it say merged instead of closed? Also, once I figure out how to edit the new squashed commit message i'll start doing that.

**Comment by tkdrg** [comment]
Yeah, this is because I manually merged it to rewrite the commit message.

**Comment by optimumtact** [comment]
you can amend the last commit you made with git commit --amend
from an appropriate cli (git bash for windows etc)
